### DISCUSSION TITLE
Заголовок панели

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#338] [Есть идея, а что если...]
Почему то не работает параметр titles в функции linkToPanel. При попытке его использовать функция совсем не вызывает панель.

**Comment by ttm-alexeypolunin** [comment]
Самая последняя версия на текущий момент должна быть — 4.9.52.6

**Reply by vldekov** [reply]
![image](https://user-images.githubusercontent.com/114401808/226827214-6fbae2a3-9b79-4cdf-b0ee-7cdeec19f956.png)

**Reply by ttm-alexeypolunin** [reply]
titles могут быть использованы двумя путями.

Как row:

```
= : linkToPanel(table: 'prostaya'; titles: $row; id: #h_id; refresh: false; columns: 1)

row: rowCreate(field: "select_1" = "Проверка 1"; field: "select_2" = "Проверка 2")
```

Как rowlist:

```
= : linkToPanel(table: 'prostaya'; titles: $rowList; id: #h_id; refresh: false; columns: 1)

rowList: rowListCreate(field: "name" = $listOfNames; field: "title" = $listOfTitles)

listOfNames: listCreate(item: "select_1"; item: "select_2")
listOfTitles: listCreate(item: "Проверка 1"; item: "Проверка 2")
```

**Reply by vldekov** [reply]
Я думал что titles это текст заголовка панели. Как это сделано в LinkToTable ( title —  заголовок, отображаемый в открытом окне для iframe ). Есть возможность изменить текст заголовка в панели?

**Reply by ttm-alexeypolunin** [reply]
Не делали, надо добавить.
